optaplanner-examples/src/test/java/org/optaplanner/examples/conferencescheduling/solver/ConferenceSchedulingScoreRulesXlsxTest.java

```diff
@@ -48,53 +52,66 @@
 import org.optaplanner.examples.conferencescheduling.domain.Timeslot;
 import org.optaplanner.examples.conferencescheduling.persistence.ConferenceSchedulingXlsxFileIO;
 import org.optaplanner.test.impl.score.buildin.hardsoft.HardSoftScoreVerifier;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import static org.optaplanner.examples.common.persistence.AbstractXlsxSolutionFileIO.DAY_FORMATTER;
 import static org.optaplanner.examples.common.persistence.AbstractXlsxSolutionFileIO.TIME_FORMATTER;
 
+@RunWith(Parameterized.class)
 public class ConferenceSchedulingScoreRulesXlsxTest {
 
-    private final String testFileName = "testConferenceSchedulingScoreRules.xlsx";
-    private final HardSoftScore unassignedScore = HardSoftScore.ZERO;
-    private final transient Logger logger = LoggerFactory.getLogger(getClass());
+    private static final String testFileName = "testConferenceSchedulingScoreRules.xlsx";
+    private static final HardSoftScore unassignedScore = HardSoftScore.ZERO;
 
-    private ConferenceSolution initialSolution;
-    private String currentPackage;
-    private String currentConstraint;
+    private String constraintPackage;
+    private String constraintName;
     private HardSoftScore expectedScore;
+    private ConferenceSolution solution;
+    private String testSheetName;
 
-    private HardSoftScoreVerifier<ConferenceSolution> scoreVerifier = new HardSoftScoreVerifier<>(
+    private static HardSoftScoreVerifier<ConferenceSolution> scoreVerifier = new HardSoftScoreVerifier<>(
             SolverFactory.createFromXmlResource(ConferenceSchedulingApp.SOLVER_CONFIG));
-    private TestConferenceSchedulingScoreRulesReader testFileReader;
 
-    @Before
-    public void setup() {
-        File testFile = new File(getClass().getResource(testFileName).getFile());
+    public ConferenceSchedulingScoreRulesXlsxTest(String constraintPackage, String constraintName,
+                                                  HardSoftScore expectedScore, ConferenceSolution solution, String testSheetName) {
+        this.constraintPackage = constraintPackage;
+        this.constraintName = constraintName;
+        this.expectedScore = expectedScore;
+        this.solution = solution;
+        this.testSheetName = testSheetName;
+    }
+
+    @Parameterized.Parameters(name = "{4}")
+    public static Collection testSheetParameters() {
+        List<Object[]> parametersList = new ArrayList<>();
+        Object[] currentParameterList;
```

Comment:
This is just a nitpicking to improve readability a bit - the `currentParameterList` could be declared right before the while loop, as this is the place where it's used for the first time.